GitHubAssignees.ps1

```diff
@@ -379,6 +389,11 @@ function Remove-GithubAssignee
         'assignees' = $Assignee
     }
 
+    if ($Force -and -not $Confirm)
```

Comment:
I always prefer to be quite explicit with my grouping of expressions and make not assumption on the language (or reader's) understanding of priority:

```suggestion
    if ($Force -and (-not $Confirm))
```

(here and elsewhere)

Reply:
Fixed 😄